pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -74,6 +73,14 @@ def configure_checkpoint_callbacks(self, checkpoint_callback: Union[ModelCheckpo
             self.trainer.callbacks.append(ModelCheckpoint(dirpath=None, filename=None))
 
     def configure_progress_bar(self, refresh_rate=1, process_position=0):
+        # smaller refresh rate on colab causes crashes, warn user about this
+        if IS_COLAB and refresh_rate < 20:
```

Comment:
```suggestion
        if os.getenv('COLAB_GPU') and refresh_rate < 20:
```
I suggest using this since this is what we use at other places.

Reply:
Does this work for Colab CPU and TPU runtimes too?

Reply:
Yes